examples/react-example/src/ui/App.tsx

```diff
@@ -135,21 +146,23 @@ export const App: React.FC = () => {
           Transfer
         </button>
       </div>
-
-      <LinkComponent
-        linkToken={linkToken}
-        onIntegrationConnected={(authData: LinkPayload) => {
-          setPayload(authData)
-          setLinkToken(null)
-        }}
-        onExit={err => {
-          setLinkToken(null)
-          setError(err || null)
-        }}
-        onTransferFinished={data => {
-          setTransferFinishedData(data)
-        }}
-      />
+      <WagmiCoreProvider>
```

Comment:
lets not force user to add more dependencies than needed.  link should be started by  `createLink` and `openLink` only